featureHome/src/test/kotlin/com/tapptitude/featurehome/fake/FakeSessionManager.kt

```diff
@@ -0,0 +1,30 @@
+package com.tapptitude.featurehome.fake
+
+import com.tapptitude.session.SessionManager
+import com.tapptitude.session.model.LoggedIn
+import com.tapptitude.session.model.LoggedOut
+import com.tapptitude.session.model.LoginState
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+internal class FakeSessionManager : SessionManager {
+    private val _currentLoginStateFlow: MutableStateFlow<LoginState> = MutableStateFlow(
```

Comment:
What's the `Hard Wrap` setting value in your Android Studio? I think anything under ~120 is a bit too short